src/types/serialization.js

```diff
@@ -20,7 +20,7 @@ export function serialize(buffer, shift, data, type) {
                 if (!isFixed(fields[fieldName].type.fields)) {
                     return false;
                 }
-            } else if (fields[fieldName].type === String) {
+            } else if (fields[fieldName].type === String || fields[fieldName].type === FixedBuffer) {
```

Comment:
Why though? `FixedBufer`s have fixed length by definition, as far as I understand.

Reply:
Thats because `FixedBuffer` is serialized as segment in Exonum core.